src/webview/create-component/pages/fromExistingGitRepo.tsx

```diff
@@ -336,7 +339,7 @@ export function FromExistingGitRepo({ setCurrentView }) {
                                     </Stack>
                                 )}
                             </>
-                        ) : recommendedDevfile.noRecommendation && !selectedDevfile ? (
+                        ) : recommendedDevfile.noRecommendation && !selectedDevfile? (
```

Comment:
I think removing the space here is misleading

I'm pretty sure the implicit order of operations is:

```
) : (recommendedDevfile.noRecommendation && !selectedDevfile)? (
```

Reply:
Ack